### PR TITLE
Fabo/fix addres

### PR DESCRIPTION
**Pull request by faboweb**
Mostly for @jbibla showing correct addresses in tests

### CODE REVIEW

File: tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap [hunk 1]
```diff
@@ -132,7 +132,7 @@ exports[`TmSessionExplore shows a form to sign in with an address 1`] = `
               class="tm-li-session-title"
             >
               <span>
-                cosmos1vxkye…a7m64khj8xyz
+                cosmos15ky9d…rm5ctpesxxn9
```
Comment (jbibla) [inline tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: ?

Comment (Bitcoinera) [reply tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: why did we change the format to a much longer one?

Comment (faboweb) [reply tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: ups, this is still not a valid bech32 address

File: tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap [hunk 2]
```diff
@@ -132,7 +132,7 @@ exports[`TmSessionExplore shows a form to sign in with an address 1`] = `
               class="tm-li-session-title"
             >
               <span>
-                cosmos15ky9d…rm5ctpesxxn9
+                cosmos…gc05sslwesuy
```
Comment (Bitcoinera) [inline tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: I think it is better like it was before. Showing four at the beginning, after the `cosmos` prefix, and four at the end.

Like this is much easier to identify imo

Comment (faboweb) [reply tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: would be a change though. cosmos…gc05sslwesuy is how we displayed it all the time

**Pull request merged**